server/src/main/scala/za/co/absa/atum/server/model/CheckpointItemFromDB.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.server.model
+
+import io.circe.Json
+
+import java.time.ZonedDateTime
+import java.util.UUID
+
+case class CheckpointItemFromDB(
```

Comment:
the same thing about the placement of these 'fromDB data containers' can be applied here as well :)

Reply:
As above I would rather argue for a separate package as again multiple fa-db object could be returning the same data structure.